consensus/raft/raft.go

```diff
@@ -108,15 +108,15 @@ func newRaftWrapper(peers []peer.ID, host host.Host, cfg *Config, fsm hraft.FSM)
 		return nil, err
 	}
 	if !hasState {
-		logger.Info("bootstrapping raft cluster")
+		logger.Info("initializing raft cluster")
 		err := hraft.BootstrapCluster(cfg.RaftConfig,
 			log, stable, snap, transport, srvCfg)
 		if err != nil {
 			logger.Error("bootstrapping cluster: ", err)
 			return nil, err
 		}
 	} else {
-		logger.Info("raft cluster is already bootstrapped")
+		logger.Debug("raft cluster is already bootstrapped")
```

Comment:
As you changed bootstrapping to initializing above it seems like you should do it here too for consistency.